Source/Meadow.ProjectLab/ProjectLabHardwareV3.cs

```diff
@@ -360,4 +360,27 @@ public override ModbusRtuClient GetModbusRtuClient(int baudRate = 19200, int dat
     {
         return _connectors.GetModbusRtuClient(this, baudRate, dataBits, parity, stopBits);
     }
+
+    /// <inheritdoc/>
+    public override ITouchScreen? Touchscreen
```

Comment:
We should move this into a v4 class but great to leave it for now

Reply:
The hardware I have with a touchscreen is 3.f, so.....

Reply:
@ctacke Bryan confirmed he's bumped the PL with touchscreen to v4, it'll show up in the upcoming board revisions.